02_configure_host.sh

```diff
@@ -312,6 +312,16 @@ mkdir -p "${M3PATH}"
 detect_mismatch "${BMO_LOCAL_IMAGE:-}" "${BMOPATH}"
 clone_repo "${BMOREPO}" "${BMOBRANCH}" "${BMOPATH}" "${BMOCOMMIT}"
 
+export DEPLOY_TOOL="${BMOPATH}/tools/deploy.sh"
+if [[ "${BMOBRANCH}" = "main" ]] || [[ "${BMOBRANCH}" = "release-0.9" ]]; then
```

Comment:
Since deploy.sh is going away, make it the subject of condition, so we don't have to be adding releases in this list to keep it working.

```suggestion
if [[ "${BMOBRANCH}" =~ release-0.(8|6|5) ]]; then
    export DEPLOY_TOOL="${BMOPATH}/tools/deploy.sh"
else
```

Reply:
Now it is in two places?